### PR TITLE
DEV-2305: First Pass Refactoring "Learn"

### PR DESCRIPTION
**Pull request by milldr**
## what
- Refactor all pages and chapters under "Learn"

## why
- Clean up all files in this section for our updated guidelines

## references
- DEV-2381

### CODE REVIEW

File: docs/layers/accounts/tutorials/how-to-set-up-aws-email-notifications.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+# How to Set Up AWS Email Notifications
+
+:::caution Make sure to use the email address corresponding to the
+[Decide on Email Address Format for AWS Accounts](/reference-architecture/fundamentals/design-decisions/cold-start/decide-on-email-address-format-for-aws-accounts)
+ADR.
+
+:::
+
```
Comment (osterman) [inline docs/layers/accounts/tutorials/how-to-set-up-aws-email-notifications.md:15]: Weird admonition. The title is like a sentence.

File: docs/layers/accounts/tutorials/terraform-s3-state.md [hunk 2]
```diff
@@ -101,8 +114,10 @@ aws --profile acme-gbl-root-admin \
   s3 ls s3://{bucket_name}/vpc/
 ```
 
-:::note
-If a component is mistakenly deployed somewhere and destroyed, a leftover `terraform.tfstate` file will be present on your local filesystem with a small file size so while this is a good way to search for backends, it's not the best way to determine where a component is deployed. Also, the S3 bucket has versioning enabled, ensuring we can always (manually) revert to a previous state if need be.
+:::note If a component is mistakenly deployed somewhere and destroyed, a leftover `terraform.tfstate` file will be
```
Comment (osterman) [inline docs/layers/accounts/tutorials/terraform-s3-state.md:117]: Why are you removing the newline?

Comment (milldr) [reply docs/layers/accounts/tutorials/terraform-s3-state.md:117]: It was formatted with prettier. I'll fix this

File: docs/layers/accounts/tutorials/how-to-set-up-aws-email-notifications.md (hunk 1 above)
Comment (milldr) [reply docs/layers/accounts/tutorials/how-to-set-up-aws-email-notifications.md:15]: prettier formatted these incorrectly. I'll fix

**Review by osterman — APPROVED** [review]

**Pull request merged**